web/backend/index.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express')
+const app = express()
+
+app.get('/', function (req, res) {
+    res.send('Hello World')
+})
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/42ISTblockchain/business-basket-app/pullRequest?prid=9595197)
```suggestion
});
```